perma_web/functional_tests/test_links.py

```diff
@@ -2,34 +2,56 @@
 
 two_minutes = 120 * 1000
 
-
-def test_create_link(logged_in_user) -> None:
-    """It should be possible to successfully create a link from a URL"""
-    url_field = logged_in_user.locator('#rawUrl')
+def create_link(page):
```

Comment:
Is this basically a fixture/helper for the `test_create_link_XXXX_playback` tests? The roles of the test functions here read a little ambiguously to me. Not a blocker—I think it's a matter of preference.

Reply:
> Is this basically a fixture/helper
Yep, just a helper for making a Perma Link.

Hmmm, how can I make this clearer... I'm not sure a single parametrized test function is the way to go, since the assertions are opposites. I think this is awkward because it is simply weird that one user would see a WARC playback and one would see a WACZ playback, just because of who they are (rather than something in the UI), but since that's how this feature flag works for now, it's just awkward.

Reply:
Makes sense, thanks!

Reply:
> Hmmm, how can I make this clearer... 

Gee, Cremona, how about you start by not leaving exactly the same cut-paste doc string for both tests 🙄 🤣 

I will at least update these docstrings.

Reply:
Added new docstrings for all three 👍